controller.go

```diff
@@ -217,13 +225,13 @@ func main() {
 		awsAdapter.NewIAMCertificateProvider(),
 	)
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
```

Comment:
This should Fatal, because of restarting automatically and trying to get AWS credentials from cache again